### PR TITLE
fix: add conversation max width, fix video, image, link preview display

### PR DESCRIPTION
**Pull request by arjita-mitra**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - add conversation max width, fix video, image, link preview display

- The **PR Description**
  - fix messages alignment issues
----

### CODE REVIEW

File: src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageActions.styles.ts [hunk 1]
```diff
@@ -106,5 +106,5 @@ export const getActionsMenuCSS = (isActive?: boolean): CSSObject => {
 };
 
 export const messageWithHeaderTop: CSSObject = {
-  top: '45px',
+  top: '-53px',
```
Comment (atomrc) [inline src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageActions.styles.ts:109]: This is a little worrying, it there an explanation for this value?

File: src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx [hunk 2]
```diff
@@ -169,7 +169,7 @@ const FailedToAddUsersMessage: React.FC<FailedToAddUsersMessageProps> = ({
           />
         </p>
       </div>
-      <div className="message-body">
+      <div className="message-body" style={{flexDirection: 'column'}}>
```
Comment (atomrc) [inline src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx]:
```suggestion
      <div className="message-body" css={{flexDirection: 'column'}}>
```

File: src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageActions.styles.ts (hunk 1 above)
Comment (arjita-mitra) [reply src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageActions.styles.ts:109]: this is to position the reaction floating menu with rest to the message header

**Comment by atomrc** [comment]
Seems like a glitch here
image

**Comment by atomrc** [comment]
Small images' read-receipt are not aligned with other read-receipts
image

**Comment by atomrc** [comment]
Read-receipt for images doesn't seem to align with the top of the image (is it by design)? 
image

**Review by atomrc — APPROVED** [review]

**Review by V-Gira — APPROVED** [review]

**Pull request merged**